### ISSUE TITLE
Some broken examples

### ISSUE THREAD

**Issue by sbryngelson** [MFlowCode/MFC#98]
Many of the `example/case.py` files do not run to completion. @js-spratt this occurs during post process for the hypoelastic cases.

@anshgupta1234 some other cases have problems like `0D_QBMM`. Can you find all such cases in `examples`, list them here, then test to see if they ran on a much older version of the code (to be sure no bugs were introduced)? Perhaps going back all the way to https://github.com/MFlowCode/MFC/releases/tag/v4.0.0

**Comment by sbryngelson** [comment]
Also: 
```
Some example cases have CFL errors before completing. For example, 2d-shock-bubble and 3d-sph-bubble-collapse.
```

**Comment by sbryngelson** [comment]
This issue is now fixed except for the post-processing bugs in hypoelasticity.

**Comment by sbryngelson** [comment]
Closed with #107
